src/modules/location/sagas.ts

```diff
@@ -2,9 +2,14 @@ import { all, takeLatest, put, select } from 'redux-saga/effects'
 import { getLocation, replace } from 'connected-react-router'
 import { locations } from 'routing/locations'
 import { LOGIN_SUCCESS, LoginSuccessAction } from 'modules/identity/actions'
+import { CLAIM_NAME_SUCCESS, SetENSContentSuccessAction, SET_ENS_CONTENT_SUCCESS } from 'modules/ens/actions'
 
 export function* locationSaga() {
-  yield all([takeLatest(LOGIN_SUCCESS, handleLoginSuccess)])
+  yield all([
+    takeLatest(LOGIN_SUCCESS, handleLoginSuccess),
+    takeLatest(SET_ENS_CONTENT_SUCCESS, handleSetENSContentSuccess),
+    takeLatest(CLAIM_NAME_SUCCESS, goToActivity)
```

Comment:
Didn't put both SET_ENS_CONTENT_SUCCESS and CLAIM_NAME_SUCCESS because the former does a validation first.
on the future we could have 
```ts
takeLatest([CLAIM_NAME_SUCCESS,...otherActions], goToActivity)
```
To redirect to activity